CONTRIBUTING.md

```diff
@@ -6,11 +6,11 @@
 
         $ git checkout -b <branch>
 
-3.  If adding a function `R.foo`, define it in __src/foo.js__, require it in
+3.  If adding a function `R.foo`, define it in __source/foo.js__, require it in
     __index.js__, and include unit tests in __test/foo.js__. It is not necessary
```

Comment:
`index.js` has also moved, to `source/index.js`, and we now use es6 modules, so it should be `export`ed rather than `require`d